src/src/LowPassFilter.h

```diff
@@ -44,9 +44,11 @@ class LPF
 
     void init(int32_t Indata)
     {
-        for (int i = 0; i < 255; i++)
-        {
-            this->update(Indata);
-        }
+        SmoothDataINT = Indata;
+        SmoothDataFP = SmoothDataINT << FP_Shift;
+        // for (int i = 0; i < 50; i++)
+        // {
+        //     this->update(Indata);
```

Comment:
Do we need this? No longer a need to call `update()`?

Reply:
Yup, I just set the internal states directly